lib/identity_cache/query_api.rb

```diff
@@ -120,7 +120,14 @@ def check_association_scope(association_name)
         end
       end
 
-      def record_from_coder(coder) #:nodoc:
+      def record_from_coder(coder)
+        return unless coder
+        ActiveSupport::Notifications.instrument('identity_cache.hydration', class: coder[:class]) do
```

Comment:
i.e. serialization actually happens within the `MemcachedStore`layer (iirc) where the `Hash` gets serialized into a byte array. This process _is not part_ of this measurement